### PR TITLE
migrated javascript file to typescript (Interests.js to Interests.tsx)

### PR DESCRIPTION
**Pull request by shubhangi013**
**Fixes**

Fixes part of #93 by @subodhk01

**Description**

Migration of Interests.js to Interests.tsx

**Checklist:**

-   [x] I have read the contribution guidelines in the repository.
-   [x] My branch is up-to-date with upstream branch.
-   [x] My pull request has a descriptive title (not a vague title like "Update package.json").
-   [x] My code follows the established code style of the repository.
-   [x] I have tried running the project locally and verified that there are no visible errors.

### CODE REVIEW

**Comment by tiger-yash** [comment]
You have linked the PR to the wrong issue @shubhangi013

**Comment by shubhangi013** [comment]
> You have linked the PR to the wrong issue @shubhangi013

yeah my bad fixed it

File: components/Profile/Interests.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import { Text, Tag } from "atomize"
 
-export const Interests = ({ interests }) => {
-    const skills = interests.length
-        ? interests.split(",").map((skill, id) => {
-              return (
+export const Interests = ({ interests}:{interests : string}) => {
+    const skills :JSX.Element[] = interests.length
```
Comment (shubhanshu02) [inline components/Profile/Interests.tsx:4]: This line shows an ESLint error as follows:
>  4:19  error  'JSX' is not defined  (no-undef)

To fix this, add the `JSX` to the `globals` in the ESLint configuration file `.eslint.js` as follows:
```js
{
    ... // the lines above
    globals: {
        React: "writable",
        JSX: true,
    },
}
```

**Review by shubhanshu02 — CHANGES_REQUESTED** [review]
Just a fix, rest everything looks good. Also, rebase your branch with the current master and use prettier before committing changes.

File: components/Profile/Interests.tsx (hunk 1 above)
Comment (shubhangi013) [reply components/Profile/Interests.tsx]: Thanks. Fixed it.

Comment (shubhanshu02) [reply components/Profile/Interests.tsx]: Great! Just one thing left, merge the changes in this repo to your branch or rebase using `git pull upstream main --rebase`. If you use rebase, you'll need to force push the changes.

Comment (shubhangi013) [reply components/Profile/Interests.tsx]: Yeah merged the changes.

**Review by shubhanshu02 — APPROVED** [review]
Thanks, @shubhangi013 :tada:

**Pull request merged**